corehq/ex-submodules/couchexport/writers.py

```diff
@@ -390,10 +398,11 @@ def get_write_value(value):
                 value = ''
             return dirty_chars.sub('?', value)
 
-        # NOTE: don't touch this. changing anything like formatting in the
```

Comment:
How come this comment doesn't apply anymore?

Reply:
After upgrading openpyxl, we now open the workbook with the kwarg ```write_only=True```.  This prevents us from introducing the types of memory issues that are warned against here.  I moved the link to the documentation explaining why ```write_only=True``` is used to the line where we open the workbook.